lyteCache.php

```diff
@@ -177,6 +177,15 @@ function lyte_get_thumb( $thumbUrl ) {
         curl_setopt( $curl, CURLOPT_CONNECTTIMEOUT, 5 );
         $str = curl_exec( $curl );
         $err = curl_error( $curl );
+        
+        // fallback to hqdefault.jpg if maxresdefault.jpg is missing
+        $info = curl_getinfo($curl);
+        if ($info['http_code'] === 404) {
```

Comment:
Can you add an extra condition here to make sure `$thumbUrl` does contain `maxresdefault.jpg` to avoid not replacing anything and doing a curl-request for the same resource?

Reply:
Hi, good idea. The code now checks for maxresdefault.jpg in $thumbUrl